rid/v1/commons.yaml

```diff
@@ -0,0 +1,107 @@
+openapi: 3.2.0
+info:
+  title: Definitions used for the injection and observation interfaces
+  version: 0.1.0
+components:
+  schemas:
+    Altitude:
+      format: float
+      description: 'Geodetic altitude (NOT altitude above launch, altitude above
```

Comment:
I think we want to transition away from language that's specifically aligned with ASTM standard text -- for instance, the USS should provide whatever altitude they know without regard to resolution.  When we perform ASTM F3411 tests, we'll check whether the minimum resolution requirement is met, but that might not be a requirement in other RID regimes.  For that reason, I think I actually like the old description better ("An altitude, in meters, above the WGS84 ellipsoid.")  Also, we don't want to mention "aircraft" since it's use in OperatorAltitude also.